### PR TITLE
change password functionalities implemented

### LINKED ISSUE #936: Reinstate `Change Password` button
We've lost the `Change Password` from the top-right context menu.

We should put it back.

### PR DESCRIPTION
**Pull request by TahaKhanAbdalli**
fixes #936

### CODE REVIEW

File: _devExtensions/changePassword-controller.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+const bcrypt = require('bcryptjs')
+const BS3Database = require('better-sqlite3')
+const path = require('path')
+const passwordValidationSchema = require('./password-validation.schema')
+const tableName = 'passwords'
+
+const updateUserPassword = (db, userId, password) => {
+  console.log(userId, password)
```
Comment (IanMayo) [inline _devExtensions/changePassword-controller.js:8]: we shouldn't have this logging line, should we?

**Comment by IanMayo** [comment]
Codacy isn't happy @TahaKhanAbdalli ;-)

image

**Comment by IanMayo** [comment]
New problem:
image

(I'm posting the images @TahaKhanAbdalli , because I don't know if these errors get shown to you in GitHib)

**Comment by TahaKhanAbdalli** [comment]
> New problem: image
> 
> (I'm posting the images @TahaKhanAbdalli , because I don't know if these errors get shown to you in GitHib)

working on it

**Pull request merged**